api/tests/test_cloud_record_summary.py

```diff
@@ -18,8 +15,134 @@ class CloudRecordSummaryTest(TestCase):
     """Tests GET requests to the Cloud Sumamry Record endpoint."""
 
     def setUp(self):
-        """Disable logging.INFO from appearing in test output."""
-        logging.disable(logging.INFO)
+        """Prevent logging from appearing in test output."""
```

Comment:
looks like we could do this with django-nose, but not with the version of django we are using

http://www.marinamele.com/2014/03/tools-for-testing-in-django-nose.html